camera.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2025 PHYTEC Messtechnik GmbH
+# SPDX-License-Identifier: Apache-2.0
+# Author: Martin Schwan <m.schwan@phytec.de>
+
+import cv2 as cv
```

Comment:
If doing it new, we should organzied the imports correctly from scratch: standard imports first, the OpenCV.